### DISCUSSION TITLE
Set default poll language

### DISCUSSION THREAD

**Discussion by MisterT2** [lukevella/rallly#367] [Ideas]
Could it be possible to set the default language when building the poll ?
(of course, users may change "manually" in their options later)

**Comment by lukevella** [comment]
The language the user sees depends on what languages the user’s browser is configured to accept. This is pretty standard behavior across many websites and gives the best possible outcome in my opinion, as everyone sees the page in their desired language. If there’s some argument for changing this behavior I’d be interested to hear about it.

**Reply by MisterT2** [reply]
Oh !
I was not aware of that implementation. pinciple.
I agree it is the one that makes the most sense, even if I know may people that have never configured their browser language preferences...
Let's continue like this then...
Thanks for this reply and this excellent tool !
